tools/bind.sh

```diff
@@ -23,16 +23,18 @@ Run bind or unbind operations
 Usage: $0 [-p] COMMAND
 
 Commands:
-    bind            bind all drives to the rabbit
-    unbind          unbind all drives from the rabbit
+    bind   [PHYSICAL_PORT] [VIRTUAL_FUNCTION]   bind all drives to the PHYSICAL_PORT at VIRTUAL_FUNCTION.   Default: bind to Rabbit-P(PP:24) @ (VF:1)
```

Comment:
Is this still incorrect and will only bind the local 9 drives? If so, I hesitate to add it. Or add a huge WARNING! message.

Remember we share these scripts with IST and if they see this as you currently have it, they will be frustrated when it doesn't work as advertised.

Reply:
It is incorrect for anything other than Rabbit, yes. I will remove this file for now since I don't think it does much other than confuse things.